src/client/java/teammates/client/scripts/scalabilitytests/InstructorCourseEnrollPageScaleTest.java

```diff
@@ -0,0 +1,94 @@
+package teammates.client.scripts.scalabilitytests;
+
+import org.testng.annotations.Test;
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.common.util.Logger;
+import teammates.client.scripts.util.Stopwatch;
+import teammates.test.cases.browsertests.BaseUiTestCase;
+import teammates.test.pageobjects.InstructorCourseEnrollPage;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Covers 'enroll' view for instructors. Run 'instructorCourseEnrollPageScaleTestDataGenerator.py' before running the tests.
+ * SUT: {@link InstructorCourseEnrollPage}.
+ */
+public class InstructorCourseEnrollPageScaleTest extends BaseUiTestCase {
+
+    private static final String DATA_FOLDER_PATH = "src/client/java/teammates/client/scripts/scalabilitytests/";
+    private InstructorCourseEnrollPage enrollPage;
+    private Logger logger = Logger.getLogger();
```

Comment:
Loggers should be static and final, and name it `log` to standardize.

Alternatively, do we really need a logger? Will `System.out.println` work?